crates/telio-dns/src/nameserver.rs

```diff
@@ -157,7 +157,11 @@ impl LocalNameServer {
                         {
                             Ok(length) => length,
                             Err(e) => {
-                                telio_log_error!("[DNS] {}", e);
+                                telio_log_error!(
+                                    "[DNS] {}. Offending request packet: {:?}",
+                                    e,
+                                    packet
```

Comment:
what is most pessimistic scenario in the context of packet size (length) and how often it can be called?

Reply:
worst case scenario - 1420 bytes, called on every DNS request. It is rather verbose, but we are specifically trying to why this error is occurring.